docs/cypress-tutorial.md

```
@@ -0,0 +1,844 @@
+cd frontend
+export CY_TEST_CONFIG=/path/to/test-variables.yml
+npm run cypress:open
+```
+
+1. Select **E2E Testing** → **Chrome**
+2. Open `cypress/tests/e2e/dataScienceProjects/testProjectCreation.cy.ts`
+3. Run the test - it should pass
+4. Click on `findCreateProjectButton` command to see the DOM snapshot
+5. Press **F12** → inspect the `data-testid="create-project"` attribute
+
+**Step 4: Change and update the test ID**
```

markdownlint-cli2 (0.20.0)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)